imports/ui/pages/profile.jsx

```diff
@@ -50,7 +50,7 @@ class _Profile extends Component {
       const UID = UUID.v5({
         namespace: '00000000-0000-0000-0000-000000000000',
         name: fileURL})
-      let image = {UID: UID}
+      let image = {UID: UID, url: fileURL}
```

Comment:
But don't the lines below (if existing) then save the existing image to both the profile and thumbnail?

Also, isn't fileURL actually the base64 data, so not what we want to search based on, as it's not stored and would never be found?